src/js/personalization/va-profile-beta/components/AddressSection.jsx

```diff
@@ -51,21 +62,45 @@ class EditAddressModal extends React.Component {
   }
 }
 
-export default function AddressSection({ addressResponseData, title, field, error, clearErrors, isEditing, isLoading, onChange, onEdit, onCancel, onSubmit }) {
+function AddressView({ address }) {
+  const street = [
+    address.addressOne,
+    address.addressTwo ? `, ${address.addressTwo}` : '',
+    address.addressThree ? ` ${address.addressThree}` : ''
+  ].join('');
+
+  const country = address.type === ADDRESS_TYPES.international ? address.countryName : '';
+  let cityStateZip = '';
+
+  switch (address.type) {
+    case ADDRESS_TYPES.domestic:
+      cityStateZip = `${address.city}, ${getStateName(address.state)} ${address.zipCode}`;
+      break;
+    case ADDRESS_TYPES.military:
+      cityStateZip = `${address.militaryPostOfficeTypeCode}, ${address.militaryStateCode} ${address.zipCode}`;
+      break;
+    case ADDRESS_TYPES.international:
+    default:
+      cityStateZip = address.city;
+  }
+
+  return (
+    <div>
+      {street}<br/>
+      {cityStateZip}<br/>
+      {country}
+    </div>
+  );
+}
+
```

Comment:
@bshyong there is no consolidation with the address formatting here, fyi. That's only done when going into the update-modal. It was too confusing for too little return to try to consolidate it everywhere.